recipes/fusion-filter/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "fusion-filter" %}
+{% set version = "0.5.0" %}
+{% set build = "0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source: 
+  fn: FusionFilter-v{{ version }}.tar.gz
+  url: https://github.com/FusionFilter/FusionFilter/archive/FusionFilterv{{ version }}.tar.gz
+  md5: c7c3030188ae56499d6222ece5dd682f
+
+build:
+  number: {{ build }}
+  skip: true  # [py3k]
+
+requirements:
+  build:
+  run:
+    - python
+    - perl
+    - perl-perlio-gzip
+    - perl-uri
+    - perl-set-intervaltree
+    - perl-db-file
+    - perl-carp
+    - perl-json-xs
+    - samtools
+    - gmap >=2017.10.30
+    - star >=2.5.4a
+    - blast >=2.5.0
```

Comment:
You may want to put `>=2.7.1` - otherwise you'll get the 2.6 due to issues with `conda`

Reply:
Hello. I am not sure whether you mean I should set  >=2.7.1 for python or for blast.

Reply:
But actually, I don't know if FusionFilter will work with releases of blast that are greater than 2.5.0. The release states that it uses 2.5.0, so perhaps I should pin it to that release for now.

Reply:
I meant blast, of course :). I do have an upstream issue at conda/conda#7112 - conda installing an older blast is regarded a bug. You should definitely test whether fusion works with blast 2.7.1. If it doesn't, create an issue upstream at fusion and pin for the time being. I spent quite a bit of time on the 2.7.1 package to build blast from source, saving several 100MB installed size (downloaded size quickly adds up and becomes an issue for those of us testing their tools on CI).

Reply:
OK. So I will be setting the Blast version to be >=2.7.1. Also have made the other changes you requested, which I will be pushing up.